source/tablist.rst

```diff
@@ -0,0 +1,33 @@
+==================
+PlayerList/Tablist
+==================
+
+Adventure only supports changing the header(encased in red) and footer(encased in yellow) of the tablist.
```

Comment:
```suggestion
Adventure only supports changing the header (encased in red) and footer (encased in yellow) of the tablist.
```

any chance you could use thicker lines for those? and maybe make one dashed and one solid, just so we don't require colour for readers to understand what you're saying.

Reply:
Thats a good idea. I forgot about color blindness